packages/client/mutations/CreateJiraTaskIntegrationMutation.ts

```diff
@@ -53,52 +52,6 @@ const CreateJiraTaskIntegrationMutation: StandardMutation<TCreateJiraTaskIntegra
   return commitMutation<TCreateJiraTaskIntegrationMutation>(atmosphere, {
     mutation,
     variables,
-    updater: (store) => {
-      // TODO break out into subscription & also reorder suggested items (put newest on top if exists)
-      const payload = store.getRootField('createJiraTaskIntegration')
-      if (!payload) return
-      const task = payload.getLinkedRecord('task')
-      if (!task) return
-      const userId = task.getValue('userId')
-      const teamId = task.getValue('teamId')
-      const integration = task.getLinkedRecord('integration')
-      if (!userId) return
-      const user = store.get(userId)
-      if (!user || !integration) return
-      const suggestedIntegrations = user.getLinkedRecord('suggestedIntegrations', {teamId})
-      const projectKey = integration.getValue('projectKey')
-      if (!suggestedIntegrations || !projectKey) return
-      const items = suggestedIntegrations.getLinkedRecords('items')
-      if (!items) return
-      const existingIntegration = items.find(
-        (item) => item && item.getValue('projectKey') === projectKey
-      )
-      const hasMore = suggestedIntegrations.getValue('hasMore')
-      if (!existingIntegration || !hasMore) {
-        const project = integration.getLinkedRecord('project')
-        if (!project) return
-        const projectName = project.getValue('projectName')
-        const cloudId = integration.getValue('cloudId')
-        if (!projectName || !cloudId) return
-        const nextItem = {
-          cloudId,
-          projectKey,
-          projectName,
-          service: 'jira'
-        } as const
-        const id = makeSuggestedIntegrationId(nextItem)
-        // the fallback is likely never used
```

Comment:
The task successfully integrates with Jira and updates without the updater. I'm not sure how to test this fallback option or whether I should replace `suggestedIntegrationJira` with `jiraRemoteProject` here

Reply:
agreed, no updater needed for this anymore!